README.md

```diff
@@ -8,187 +8,16 @@
 [![Docs](https://img.shields.io/badge/AqNWB-Docs-8A2BE2?style=flat)](https://neurodatawithoutborders.github.io/aqnwb/)
 [![Code Stats](https://img.shields.io/badge/AqNWB-Code%20Statistics-8A2BE2?style=flat)](https://nwb-overview.readthedocs.io/en/latest/nwb-project-analytics/docs/source/code_stat_pages/AqNWB_stats.html)
 
+**AqNWB is currently under active development and should not yet be used in practice.**
 
 AqNWB is a C++ API for acquiring neurophysiological data directly into the NWB (Neurodata Without Borders) format.
 Our goal is to provide a lightweight API to integrate with existing acquisition systems.
 
-Please note, AqNWB is currently under development and should not yet be used in practice.
+Please see the [Documentation](https://neurodatawithoutborders.github.io/aqnwb/) for 
```

Comment:
For the links in the README to work both in GitHub and the Doxygen docs, I think we'll need to link to the URL on https://neurodatawithoutborders.github.io/aqnwb/ directly, rather than being able to dynamically generate the link with Doxygen as otherwise GitHub won't know what those links are.